### ISSUE TITLE
SFTP support

### ISSUE THREAD

**Issue by jcraig999** [iina/iina#2830]
Hi,
I'm trying to play a link via SFTP and I get an error every time I attempt. I noticed this open issue (https://github.com/iina/iina/issues/289), and it said something about compiling ffmpeg and the libssh Library — how exactly would I go about doing that so I can get support for SFTP? Any help would be appreciated.

**Comment by inflation** [comment]
Do you really need to go that route? If you can sftp then maybe just mounting it with sshfs. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jcraig999** [comment]
> Do you really need to go that route? If you can sftp then maybe just mounting it with sshfs.
> 
> Sent with [GitHawk](http://githawk.com)

Yeah… I have no clue what you mean. I'm not the well-versed on ffmpeg/mpv..

**Comment by chucker** [comment]
@jcraig999 this feature doesn't currently exist.

What @inflation is saying: presumably, you have some kind of SFTP client, such as Transmit. Some of them have a feature that lets you mount the remote disk into the Finder, such as [Transmit Disk](https://library.panic.com/transmit/transmit-disk/). From there, you can just open it in IINA as if it were local.
